ephios/user_management/templates/user_management/userprofile_form.html

```diff
@@ -21,10 +21,64 @@ <h1>{% trans "Add new user" %}</h1>
     </div>
     <form method="post" class="form">
         {% csrf_token %}
-        {% bootstrap_form form %}
+        {% bootstrap_form userprofile_form %}
+        {% load formset_tags %}
+
+        <div class="formset" data-formset data-formset-prefix="{{ qualification_formset.prefix }}">
```

Comment:
Can we make all qualifciations one single card and let them be list-group-items? I think that would look more apropriate, as it would section the user form better as a whole.

Reply:
also the add-button could be smaller then :joy: